### DISCUSSION TITLE
About escape in sql statements.

### DISCUSSION THREAD

**Discussion by xiaokang69** [zeromicro/go-zero#2556] [Q&A]
query := fmt.Sprintf("select %s from %s where `user_id` = ? AND `code` = ? AND `expire_time` > ? ORDER BY id DESC limit 1", verificationCodeRows, m.table)

执行sql的 会把 >号转移成 \u003e 执行sql的时候会报错

select id,user_id,code,create_time,expire_time from verification_code where user_id = 1 AND code = '582803' AND expire_time \u003e '2022-10-27 08:00:33' ORDER BY id DESC limit 1

应该怎么处理?

**Comment by xiaokang69** [comment]
帖子怎么删除呀?

**Reply by kevwan** [reply]
Let me delete it.

**Comment by kevwan** [comment]
> query := fmt.Sprintf("select %s from %s where `user_id` = ? AND `code` = ? AND `expire_time` > ? ORDER BY id DESC limit 1", verificationCodeRows, m.table)
> 
> 执行sql的 会把 >号转移成 \u003e 执行sql的时候会报错
> 
> select id,user_id,code,create_time,expire_time from verification_code where user_id = 1 AND code = '582803' AND expire_time \u003e '2022-10-27 08:00:33' ORDER BY id DESC limit 1
> 
> 应该怎么处理?

what's the problem?
